lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -102,8 +104,19 @@ function Install-Project {
 }
 set-alias install -value Install-Project
 
-Function Install-ChefMsi($msi) {
-  $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+Function Install-ChefMsi($msi, $addlocal) {
+  if ($addlocal -eq "service") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi ADDLOCAL=`"ChefServiceFeature`"" -Passthru -Wait
```

Comment:
You'll probably need to double escape the grave, but if it's more trouble that it's worth I won't be a stickler on this.